guides/micronaut-openapi-generator-server/java/src/main/java/example/micronaut/library/controller/BooksController.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Library
+ * This is a library API
+ *
+ * The version of the OpenAPI document: 1.0.0
+ * 
+ *
+ * NOTE: This class is auto generated by OpenAPI Generator (https://openapi-generator.tech).
+ * https://openapi-generator.tech
+ * Do not edit the class manually.
+ */
+
+package example.micronaut.library.controller;
+
+//tag::import[]
+import example.micronaut.library.BookService;
+//end::import[]
+import io.micronaut.http.annotation.*;
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.core.convert.format.Format;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import jakarta.inject.Inject;
+import reactor.core.publisher.Mono;
+import example.micronaut.library.model.BookInfo;
+import javax.annotation.Generated;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import javax.validation.Valid;
+import javax.validation.constraints.*;
+import io.swagger.annotations.*;
+
+@Generated(value="org.openapitools.codegen.languages.JavaMicronautServerCodegen", date="2022-01-31T13:21:29.860-05:00[America/Toronto]")
+@Controller("${context-path}")
+public class BooksController {
+
+    //tag::inject[]
+    @Inject // <1>
```

Comment:
typically @Inject and a field like this is used for dependency injection in tests, but for application code it's best to use constructor injection with a final field to make it immutable:

```
    private final BookService bookService;

    public BooksController(BookService bookService) { // <1>
        this.bookService = bookService;
    }
```